### ISSUE TITLE
Hope can provide some tips on how to generate good results like demo

### ISSUE THREAD

**Issue by visonpon** [magic-research/magic-animate#106]
Hi, thanks for sharing this wonderful work, I have test this model but the results are bad. So hope you guys can give some tips! ,thanks~

**Comment by zhangdanfeng888** [comment]
+1，我上传了自己的图片，但是生成的结果不好。

**Comment by JunbongJang** [comment]
I got better results after resizing or cropping reference image and densepose video to 512x512.

**Comment by ddpasa** [comment]
> I got better results after resizing or cropping reference image and densepose video to 512x512.

Indeed.  The readme does not make it very clear, but you should be using square videos and images.  Anything else will be severely distorted.
